config/testgrids/kubernetes/sig-storage/config.yaml

```diff
@@ -15,6 +15,7 @@ dashboard_groups:
     - sig-storage-csi-other
     - sig-storage-image-build
     - sig-storage-container-object-storage-interface-api
+    - sig-storage-container-object-storage-interface-controller
```

Comment:
Do you still need sig-storage-container-object-storage-interface-api?

Reply:
Ok.  I think this is fine.  You can probably run tests in sig-storage-container-object-storage-interface-api with a controller image, etc.